tests/test_stac.py

```diff
@@ -21,61 +21,142 @@ def test_create_collection():
 @pytest.mark.parametrize(
     "filename",
     [
-        "20220202000000-0h-enfo-ef.grib2",
-        "20220202/00z/0p4-beta/enfo/20220202000000-0h-enfo-ef.grib2",
-        (
-            "https://ai4edataeuwest.blob.core.windows.net/ecmwf/20220202/00z/"
-            "0p4-beta/enfo/20220202000000-0h-enfo-ef.grib2"
-        ),
+        "20231019000000-0h-wave-fc.grib2",
+        "20231019/00z/0p4-beta/wave/20231019000000-0h-wave-fc.grib2",
+        ("https://ai4edataeuwest.blob.core.windows.net/ecmwf/20231019/00z/"
+         "0p4-beta/wave/20231019000000-0h-wave-fc.grib2"),
     ],
 )
 def test_create_items(filename):
     item = stac.create_item_from_representative_asset(filename)
-    assert item.id == "ecmwf-2022-02-02T00-enfo-ef"
-    assert len(item.assets) == 170
+    assert item.id == "ecmwf-2023-10-19T00-wave-fc-0h"
+    assert len(item.assets) == 2
     assert item.bbox == [-180.0, -90.0, 180.0, 90.0]
-    assert item.properties["ecmwf:stream"] == "enfo"
-    assert item.properties["ecmwf:type"] == "ef"
-    assert item.properties["start_datetime"] == "2022-02-02T00:00:00Z"
-    assert item.properties["end_datetime"] == "2022-02-17T00:00:00Z"
+    assert item.properties["ecmwf:stream"] == "wave"
+    assert item.properties["ecmwf:type"] == "fc"
+    assert item.properties["ecmwf:forecast_datetime"] == "2023-10-19T00:00:00Z"
+    assert item.properties[
+        "ecmwf:reference_datetime"] == "2023-10-19T00:00:00Z"
+    assert item.properties["ecmwf:step"] == "0h"
+    assert item.properties["kerchunk_indices"] == {
```

Comment:
I'd recommend storing the expected `kerchunk_indices` in a separate `.json` file in the tests directory and reading it in for the test.

Reply:
Done.